### ISSUE TITLE
outbound emails

### ISSUE THREAD

**Issue by jonodennis1** [mypoty/poty-issues#9]
Just doing the parent set up (big list coming on that!!)

Can we have the website address on the temp password email, so we can click through......

**Comment by juur** [comment]
Done

**Comment by jonodennis1** [comment]
Just reviewing the confirmation email received from the point of registration; could it be in a html, with a nice footer and make us look super posh etc.....?

**Comment by juur** [comment]
Yes, we can, HTML formatted emails are quite an art form to get right on PC/Mobile AND not get labelled as SPAM. Currently i'm lazy and it is just converting plain text to HTML, but I can add some stylesheets etc. to tart it up a bit
